Build/TileEngine/Explosion_Control.cc

```diff
@@ -1804,24 +1804,6 @@ static void TogglePressureActionItemsInGridNo(INT16 sGridNo)
 }
 
 
-static void DelayedBillyTriggerToBlockOnExit(void)
```

Comment:
I don't think it's related. Billy does his job as expected, the problem there is that the dialog fast forwards. It has also been there since the initial import.